nncf/experimental/torch/fx/transformations.py

```diff
@@ -174,11 +207,16 @@ def constant_update_fn(model: torch.fx.GraphModule, node: torch.fx.Node, value:
 
     # Update metadata of the new constant node.
     previous_const = args[input_port_id]
-    new_constant.meta = copy(previous_const.meta)
-    new_constant.meta["val"] = value
+    consumer_nodes = list(
+        previous_const.users.keys()
```

Comment:
Please do not call `.keys` explicitly (https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/consider-iterating-dictionary.html)
```suggestion
    consumer_nodes = list(
        previous_const.users
```

Reply:
Done!